ripples/multi_session.py

```
@@ -606,193 +1320,411 @@ def plot_ripple_triggered_firing(
+    ripple: CandidateEvent,
+    sampling_rate: float | None,
+    session_id: str | None,
+    rec_ind: str | None,
+    ripple_ind: str | None,
+) -> Any:
+    raw = np.array(ripple.raw_lfp)
+    raw = raw[2000:-2000]
+    if sampling_rate:
+        sampling_rate = sampling_rate
+    else:
+        sampling_rate = 2500
```

Ruff (0.8.2)

Use ternary operator `sampling_rate = sampling_rate if sampling_rate else 2500` instead of `if`-`else`-block

Replace `if`-`else`-block with `sampling_rate = sampling_rate if sampling_rate else 2500`

(SIM108)